src/eclipse/README.md

```diff
@@ -7,21 +7,20 @@ The following points describe how to build Scala using Eclipse.
 
 0. Download the [Scala IDE bundle](http://scala-ide.org/download/sdk.html). It comes preconfigured for optimal performance.
 
-0. Run `ant init` to download some necessary jars.
-
-0. Import the project (in `src/eclipse`) via `File` → `Import Existing Projects` and navigate to `scala/src/eclipse`. Check all projects and click ok.
-
-0. You need to define a `path variable` inside Eclipse. Define `SCALA_BASEDIR` in
-`Preferences/General/Workspace/Linked Resources`. The value should be the absolute
-path to your Scala checkout. All paths in the project files are relative to this one,
-so nothing will work before you do so.
-
-  The same `SCALA_BASEDIR` variable needs to be defined as a `classpath variable` in
+0. Run `ant build` to download some necessary jars and see a successful build.
+
+0. You need to define a `path variable` and a `classpath variable` inside Eclipse, both pointing to the Scala checkout directory:
+  - (experimental): run `./update-workspace.sh scala_checkout_dir [workspace_dir]`. This should update your workspace settings
```

Comment:
Hmm, this doesn't seem to work for me:
```
$ ./update-workspace.sh  /home/mlangc/Development/git/scala /home/mlangc/Development/eclipse-workspace/mars
Using metadata directory /home/mlangc/Development/eclipse-workspace/mars and Scala checkout /home/mlangc/Development/git/scala
Couldn't find /home/mlangc/Development/eclipse-workspace/mars/.plugins/org.eclipse.core.runtime/.settings/org.eclipse.core.resources.prefs. Is /home/mlangc/Development/eclipse-workspace/mars an Eclipse workspace?
```

I was more successful with
```
$ ./update-workspace.sh  /home/mlangc/Development/git/scala /home/mlangc/Development/eclipse-workspace/mars/.metadata
```
though (notice `.metadata` at the end of the last argument).